design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -224,6 +224,52 @@ pod.
     - Mount points
     - Downward API
 
+##### Sidecar Resource Requirements / Limits
```

Comment:
This has been discussed offline and the decision was made to post it together as they are close related.
So I will repeat here my response: If there will be a a sign that there is no agreement, it will be splited.

It is also worth noting that the two topics are updating/extending an existing proposal. They are not introduced as a separate proposal.